src/build/languageServerManager.ts

```diff
@@ -1,18 +1,40 @@
-import { BaseEvent } from '../common/loggingEvents';
+import { BaseEvent, StartLanguageServerCompleted, UserSignInCompleted } from '../common/loggingEvents';
 import vscode from 'vscode';
 import { EventType } from '../common/eventType';
 import { EnvironmentController } from '../common/environmentController';
+import { LanguageServerStatus } from '../shared';
 
 export class LanguageServerManager {
+    private _languageServerStatus: LanguageServerStatus = 'Idle';
     constructor(private _environmentController: EnvironmentController) { }
 
     public eventHandler = (event: BaseEvent) => {
         switch (event.type) {
             case EventType.ExtensionActivated:
-                if (this._environmentController.userType && this._environmentController.enableAutomaticRealTimeValidation) {
+                if (this._environmentController.userType && this.readyToStartLanguageServer()) {
+                    this._languageServerStatus = 'Starting';
+                    vscode.commands.executeCommand('docs.enableRealTimeValidation');
+                }
+                break;
+            case EventType.StartLanguageServerCompleted:
```

Comment:
reconsider about that the handling logic when getting a `StartLanguageServerCompleted` with failed.